apps/wallet-mobile/src/features/AddWallet/common/LogoBanner/LogoBanner.tsx

```diff
@@ -0,0 +1,45 @@
+import {useTheme} from '@yoroi/theme'
+import React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {Space} from '../../../../components/Space/Space'
+import YoroiLogo from '../../illustrations/YoroiLogo'
+import {useStrings} from '../useStrings'
+
+export const LogoBanner = () => {
+  const strings = useStrings()
+  const {styles} = useStyles()
+
+  return (
+    <View style={styles.logoArea}>
+      <YoroiLogo />
+
+      <Text style={styles.logoName}>{strings.logoTitle}</Text>
+
+      <Space height="s" />
+
+      <Text style={styles.descriptionText}>{strings.logoSubtitle}</Text>
```

Comment:
keep consistency between the style + content, logoName + logoTitle, descriptionText + logoSubtitle, this is already a logo:
title + subtitle